plutus-core/README.md

```diff
@@ -80,26 +80,34 @@ Copied executables to ~/.local/bin:
 
 #### The CK machine
```

Comment:
I noticed that this was out of date and decided to update it since we need to include the new syntax anyway.